docs/docs/using-mdx.mdx

```diff
@@ -350,7 +351,7 @@ import type {} from 'mdx'
 // ---cut---
 import Example from './example.mdx'
 
-console.log(
+export default function Component() {
```

Comment:
I'm not sure how this change relates to the goal of improving scanning or visual structure.

Reply:
Ah yeah sorry, that was me "fixing" what I thought was a mistake (console.log would not be as common to find wrapping around JSX in a .jsx document as a function)

Reply:
`console.log` is an intentional consistent choice across the website and the readmes